From b26a60486bb31529cbb6b1cacd20657ad16dfdbd Mon Sep 17 00:00:00 2001
From: Andy Gross <agross@codeaurora.org>
Date: Tue, 31 Mar 2015 14:26:15 -0500
Subject: [PATCH 7/7] arm64: dts: qcom: Add APQ8016 SBC support

This patch set adds support for the APQ8016 SBC.

Signed-off-by: Andy Gross <agross@codeaurora.org>
[Srinivas Kandagatla]: rebased on top of kumars original patch
Signed-off-by: Srinivas Kandagatla <srinivas.kandagatla@linaro.org>
---
 arch/arm64/boot/dts/qcom/apq8016-sbc.dts | 3 ++-
 arch/arm64/boot/dts/qcom/msm8916.dtsi    | 4 +++-
 2 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/apq8016-sbc.dts b/arch/arm64/boot/dts/qcom/apq8016-sbc.dts
index 825f489..b0cac53 100644
--- a/arch/arm64/boot/dts/qcom/apq8016-sbc.dts
+++ b/arch/arm64/boot/dts/qcom/apq8016-sbc.dts
@@ -12,10 +12,11 @@
  */
 
 /dts-v1/;
-
+#include <dt-bindings/arm/qcom-ids.h>
 #include "apq8016-sbc.dtsi"
 
 / {
 	model = "Qualcomm Technologies, Inc. APQ 8016 SBC";
 	compatible = "qcom,apq8016-sbc", "qcom,apq8016", "qcom,sbc";
+	qcom,board-id = <QCOM_BRD_ID(SBC, 1, 0) QCOM_BRD_SUBTYPE_DEFAULT>;
 };
diff --git a/arch/arm64/boot/dts/qcom/msm8916.dtsi b/arch/arm64/boot/dts/qcom/msm8916.dtsi
index d369133..a7aa37d 100644
--- a/arch/arm64/boot/dts/qcom/msm8916.dtsi
+++ b/arch/arm64/boot/dts/qcom/msm8916.dtsi
@@ -22,7 +22,9 @@
 	qcom,msm-id =	<QCOM_ID_MSM8916 0>,
 			<QCOM_ID_MSM8216 0>,
 			<QCOM_ID_MSM8116 0>,
-			<QCOM_ID_MSM8616 0>;
+			<QCOM_ID_MSM8616 0>,
+			<QCOM_ID_APQ8016 0>;
+
 
 	interrupt-parent = <&intc>;
 
-- 
2.6.1

